core/filter.ts

```diff
@@ -92,14 +94,25 @@ export async function addFilterForFeed(feed: FeedValue): Promise<string> {
 }
 
 export async function deleteFilter(filterId: string): Promise<void> {
-  return deleteSyncMapById(getClient(), Filter, filterId)
+  let filter = await loadValue(Filter(filterId, getClient()))
+  let feedId = filter?.feedId
```

Comment:
Let’s just wrap this code in

```js
if (filter) {
}
```

It will be cleaner.

Reply:
Is it save to wrap `deleteSyncMapById` in `if(filter)`?
And can the filter object be deleted during `deleteSyncMapById`? 
In that cases it can be dangerous to wrap it